src/tfjs_binding.ts

```diff
@@ -15,13 +15,13 @@
  * =============================================================================
  */
 
-declare class TensorMetadata {
+export class TensorMetadata {
```

Comment:
you can keep the `declare` so you have `export declare`. This tells external tools like Closure that the object comes from an external source so it doesn't rewrite it.

Reply:
You might have to `s/class/interface`